docs/src/adr/016adr-simple.md

```diff
@@ -106,27 +106,30 @@ We can implement a specialized encoding, which does not use arena logic of any k
 <!-- Communicates what solution was decided, and it is expected to solve the
      problem. -->
 
-We propose option (2), and give the following categorization of the Simple fragment:
+We propose option (2), and give the following categorization of the rTLA fragment:
 
   - Boolean, integer and uninterpreted literals (including strings)
+  - Restricted sets: 
+    - `Int`, `Nat` or `BOOLEAN`, or 
+    - `CONSTANT`-declared and has a type `Set(T)`, for some uninterpreted type `T`, or
+    - a range `a..b`, where both `a` and `b` are in rTLA.
```

Comment:
if we don't have integers in the first version, we should also remove `a..b`, as it requires arithmetics.

Reply:
Ah, good point.